src/luarocks/cmd.tl

```diff
@@ -623,27 +621,27 @@ function cmd.run_command(description: string, commands: {string: string}, extern
 
    local args, cmdline_vars = process_cmdline_vars(...)
    local parser = get_parser(description, cmd_modules)
-   local argsp: Args = parser:parse(args) as Args --!
+   local cmdline_args: Args = parser:parse(args) as Args --!
```

Comment:
@V1K1NGbg  I meant doing the opposite replacement: 

`args` to `cmdline_args`
`argsp` to `args`